README.md

```diff
@@ -44,60 +51,72 @@ To take full advantage of the features in WP Rig, your code editor needs support
+WP Rig uses [BrowserSync](https://browsersync.io/) to enable synchronized browser testing. To take advantage of this feature, configure the `browserSync` wrapper settings in `./config/themeConfig.js` to match your local development environment. The `proxyURL` value is the URL to the live version of your local site.
 
 ### Enabling HTTPS
 In order to enable HTTPS with BrowserSync, you must supply a valid certificate and key with the Subject Alternative Name of `localhost`. If needed, WP Rig can generate a key and certificate valid for `localhost` for you with the command `npm run generateCert`.
 
 For more information, and instructions, visit the [BrowserSync wiki page](https://github.com/wprig/wprig/wiki/BrowserSync).
 
 ### gulp
-WP Rig uses a [Gulp 4](https://gulpjs.com/) build process to generate and optimize the code for the theme. All development is done in the `/dev` folder and Gulp preprocesses, transpiles, and compiles the files into the root folder. The root folder files become the active theme. WordPress ignores anything in the `/dev` folder.
+WP Rig uses a [Gulp 4](https://gulpjs.com/) build process to generate and optimize the code for the theme. All development is done in the `wp-rig` development theme. Feel free to edit any `.php` files. Asset files (CSS, JavaScript and images) are processed by gulp. You should only edit the source asset files in the following locations:
+- CSS: `assets/css/src`
+- JavaScript: `assets/js/src`
+- images: `assets/images/src`
 
-**Note:** If you have previously used Gulp, you may encounter seemingly random errors that prevent the build process from running. To fix this issue, [upgrade to Gulp 4 following the steps outlined in the WP Rig Wiki](https://github.com/wprig/wprig/wiki/Updating-to-Gulp-4).
+JavaScript source files are transpiled with [Babel](https://babeljs.io/) and CSS source files are transpiled with [PostCSS](https://postcss.org/) so that you may use modern JavaScript and CSS without worrying about browser support.
 
-JavaScript files are automatically linted using [ESLint](https://eslint.org/) in accordance with [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
+Additionally, JavaScript files are automatically linted using [ESLint](https://eslint.org/) in accordance with [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
 
 PHP and CSS files are automatically linted using [PHP CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) in accordance with [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/). To take full advantage of this setup, configure your code editor / IDE to automatically test for the WordPress Coding Standards.
```

Comment:
Need to add something about the use of stylelint in the CSS task here. Possibly break PHP and CSS into separate sections to make that clear. Off the top of my head I'm also wondering whether using both PHPCS _and_ stylelint to lint CSS is overkill. Would stylelint suffice?

Reply:
+1 on using only stylelint on CSS.